dev

```diff
@@ -141,20 +168,12 @@ docker service create \
   --env MESG_LOG_FORMAT=$MESG_LOG_FORMAT \
   --env MESG_LOG_FORCECOLORS=$MESG_LOG_FORCECOLORS \
   --env MESG_LOG_LEVEL=$MESG_LOG_LEVEL \
-  --env MESG_TENDERMINT_P2P_PERSISTENTPEERS=$MESG_TENDERMINT_P2P_PERSISTENTPEERS \
-  --env MESG_TENDERMINT_P2P_EXTERNALADDRESS=$MESG_TENDERMINT_P2P_EXTERNALADDRESS \
-  --env MESG_COSMOS_CHAINID=$MESG_COSMOS_CHAINID \
-  --env MESG_COSMOS_VALIDATORPUBKEY=$MESG_COSMOS_VALIDATORPUBKEY \
-  --env MESG_COSMOS_GENESISACCOUNT_NAME=$MESG_COSMOS_GENESISACCOUNT_NAME \
-  --env MESG_COSMOS_GENESISACCOUNT_PASSWORD=$MESG_COSMOS_GENESISACCOUNT_PASSWORD \
   --env MESG_COSMOS_GENESISACCOUNT_MNEMONIC="$MESG_COSMOS_GENESISACCOUNT_MNEMONIC" \
```

Comment:
i couldn't find a solution to also set this env variable in `DOCKER_NETWORK_FLAGS` because `MESG_COSMOS_GENESISACCOUNT_MNEMONIC` is a string with spaces..